package.json

```diff
@@ -2,8 +2,8 @@
   "scripts": {
     "test": "node_modules/.bin/_mocha",
     "prestart": "npm run theme:build",
-    "start": "NODE_ENV=production node index.js",
-    "start:dev": "NODE_ENV=development node index.js",
+    "start": "SET NODE_ENV=production & node index.js",
```

Comment:
`&` operator is used to run command in background in linux. This won't work in *nix based systems